### PR TITLE
Topbar changes

### PR DESCRIPTION
**Pull request by ozgur2dag**
Changed the navigation headers, topbars are now a part of the application.

### CODE REVIEW

**Review by zeynepsuucan — APPROVED** [review]
Looks good! Thanks

File: front-end-layer/styles/ProfileCardStyle.js
Comment (zeynepsuucan) [inline front-end-layer/styles/ProfileCardStyle.js:1]: Is this change necessary? This file is modified in both of the last 2 PR's. It will most likely create a conflict but there is very little changes made so you should be able to resolve them quickly. Only if it is not necessary you could remove them.

Comment (ozgur2dag) [reply front-end-layer/styles/ProfileCardStyle.js:1]: no it was a small bug, happening because of a missing "}". You can see both of the PR's have the same fix whatsoever.

Comment (zeynepsuucan) [reply front-end-layer/styles/ProfileCardStyle.js:1]: oh okay no problem then, resolving comment.
